perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -30,6 +53,10 @@ class GCPManagedRelationalDb(managed_relational_db.BaseManagedRelationalDb):
 
   CLOUD = providers.GCP
   SERVICE_NAME = 'managed_relational_db'
+  # These are the constants that should be specified in GCP's cloud SQL command.
+  DEFAULT_BACKUP_START_TIME = '07:00'
+  GCP_MY_SQL_VERSION = 'MYSQL_5_6'
```

Comment:
Changed to 5.7.
Mirrored implementation of AWS for `GetLatestDatabaseVersion`. Will do further implementation in next PR. 
As discussed offline, will remove backupstarttime for now and add TODO in the base class.